airflow/config_templates/airflow_local_settings.py

```diff
@@ -172,6 +196,20 @@
 
 REMOTE_LOGGING = conf.get('core', 'remote_logging')
 
+if os.environ.get('CONFIG_PROCESSOR_MANAGER_LOGGER') == 'True':
+    DEFAULT_LOGGING_CONFIG['handlers'] \
+        .update(DEFAULT_DAG_PARSING_LOGGING_CONFIG['handlers'])
+    DEFAULT_LOGGING_CONFIG['loggers'] \
+        .update(DEFAULT_DAG_PARSING_LOGGING_CONFIG['loggers'])
+
+    # Manually create log directory for processor_manager handler as RotatingFileHandler
+    # will only create file but not the directory.
+    processor_manager_handler_config = DEFAULT_DAG_PARSING_LOGGING_CONFIG['handlers'][
+        'processor_manager']
+    directory = os.path.dirname(processor_manager_handler_config['filename'])
+    if not os.path.exists(directory):
+        mkdirs(directory, 0o777)
```

Comment:
nit: constant for 777 and qq: why set 777 permission?

Reply:
My bad here, missed read the comment from [here](https://github.com/apache/incubator-airflow/blob/06584fc4b1d82a2dbba98e484d0b4515a169a818/airflow/utils/log/file_task_handler.py#L198) and thought I need 777 for the mkdirs to work. Will update it to 755.

Reply:
You are perfectly correct. Did that before but forgot about it entirely :P

Reply:
Actually our mkdirs util method already covered this... Will just `mkdirs(directory, 0o755)`